### DISCUSSION TITLE
As an admin I want to overwrite (liquid) email templates with my custom mail templates

### DISCUSSION THREAD

**Discussion by sensedrive** [directus/directus#4858] [Archive]
Hello everyone,

As far as I can see it is currently not possible to overwrite the base email templates (password reset email, etc). It is possible to add the path to the email/templates folder to the ENV variables and use this path as the base email/templates folder?

Thank you in advance!

**Comment by benhaynes** [comment]
Email Templates are on the list to become Extensions. You can follow along here: https://github.com/directus/directus/discussions/3104

**Reply by sensedrive** [reply]
Thank you very much for the quick reply. We are looking forward to use this feature in future!
